tools/clang/test/CodeGenSPIRV/spv.inline.type.enum-class.hlsl

```diff
@@ -0,0 +1,39 @@
+// RUN: %dxc -T ps_6_0 -E main -fcgl  %s -spirv | FileCheck %s
```

Comment:
Did you mean to clean up the TODOs in this test?

Reply:
Hm, this is something we should support once #5554 is fixed. Do you think it would be better to commit this with the TODOs, or to simply remove the test altogether?

Reply:
Hm, that's unfortunate. If we're going to keep the test in this PR then let's leave the TODO but uncomment the test itself and make it expected to fail, so that if/when #5554 is fixed it will get caught for an update.

Reply:
Done.